src/zope/password/zpasswd.py

```diff
@@ -134,7 +139,15 @@ def __init__(self, options):
 
     def read_input_line(self, prompt):
         # The tests replace this to make sure the right things happen.
-        return raw_input(prompt)
+        try:
+            answer = raw_input(prompt)
```

Comment:
how right you are. Will fix that later today and re-push once I've resurrected the fork.